### PR TITLE
Update vllm to 0.6.3

### PR DESCRIPTION
**Pull request by ispobock**
## Motivation

To support Qwen2-VL in https://github.com/sgl-project/sglang/pull/1546, there some dependencies need to be upgraded.

### CODE REVIEW

File: python/sglang/srt/layers/linear.py [hunk 1]
```diff
@@ -20,8 +20,10 @@
 from vllm.model_executor.layers.linear import LinearBase
```
Comment (merrymercy) [inline python/sglang/srt/layers/linear.py:20]: do we fully re-sync this file?

**Comment by ispobock** [comment]
@merrymercy @Ying1123 @zhyncs The PR tests passed. Pls help review for some patched code. It's better to trigger the nightly eval for more compatibility tests.

File: python/sglang/srt/layers/linear.py (hunk 1 above)
Comment (ispobock) [reply python/sglang/srt/layers/linear.py:20]: No, I just updated the part which is not compatible. Do we need to fully sync it?

**Comment by zhyncs** [comment]
ref https://github.com/sgl-project/sglang/actions/runs/11422948382

**Comment by zhyncs** [comment]
Nightly eval passed.

**Pull request merged**